.github/workflows/docstring_check.yaml

```diff
@@ -1,6 +1,8 @@
 name: docstring-check
+
 on:
-    push:
+    pull_request:
```

Comment:
I guess it could also be `[pull_request, push]` for the best of both worlds?

Reply:
I'm not sure keeping push adds value but it's debatable I guess. Also the jobs that list both pull_request and push will get triggered twice if you make a PR from a branch on BlueBrain remote.